content/pages/2024/speaker_briefing.md

```diff
@@ -0,0 +1,80 @@
+---
+title: EuroSciPy 2024 - Venue
+url: 2024/venue.html
+save_as: 2024/venue.html
+section: euroscipy_2024
+template: page
+slug: venue2024
+---
```

Comment:
```suggestion
---
title: EuroSciPy 2024 - Speaker Briefing
url: 2024/speaker_briefing.html
save_as: 2024/speaker_briefing.html
section: euroscipy_2024
template: page
slug: speakerbriefing2024
---
```

Reply:
I think that we need to edit the TOC such that we link to this page in the menu of the website.

Reply:
Thanks!